packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -96,6 +97,24 @@ class Scrollable extends StatefulWidget {
   }
 }
 
+// Enable Scrollable.of() to work as if ScrollableState was an inherited widget.
+// ScrollableState.build() always rebuilds its _ScrollableScope.
+class _ScrollableScope extends InheritedWidget {
+  _ScrollableScope({
+    Key key,
+    @required this.scrollable,
+    @required Widget child
+  }) : super(key: key, child: child) {
+    assert(scrollable != null);
+    assert(child != null);
+  }
+
+  final ScrollableState scrollable;
+
+  @override
+  bool updateShouldNotify(_ScrollableScope old) => true;
```

Comment:
(otherwise you'll notify too much, e.g. any time the scrollable rebuilds, for any reason at all, you'll rebuild all the customers)